.env.example

```diff
@@ -35,6 +35,7 @@ AWS_DAILY_EXPORT_SECRET_ACCESS_KEY=
 AWS_DAILY_EXPORT_BUCKET=
 AWS_BOXI_EXPORT_ACCESS_KEY_ID=
 AWS_BOXI_EXPORT_SECRET_ACCESS_KEY=
+EXPORT_SERVICE_BOXI_EXPORT_TIME=
```

Comment:
It's just a NIT. But could we move this out of the AWS env vars, and possibly into its own section with the EPR export time env var?

```
# AWS buckets
AWS_DAILY_EXPORT_ACCESS_KEY_ID=
AWS_DAILY_EXPORT_SECRET_ACCESS_KEY=
AWS_DAILY_EXPORT_BUCKET=
AWS_BOXI_EXPORT_ACCESS_KEY_ID=
AWS_BOXI_EXPORT_SECRET_ACCESS_KEY=
AWS_BOXI_EXPORT_BUCKET=

# Exports configuration
EXPORT_SERVICE_EPR_EXPORT_TIME=
EXPORT_SERVICE_BOXI_EXPORT_TIME=
```